CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to F1PyStats
+
+😄🎉 Thank you for your interest in contributing to this repository 😄🎉 
+
+If you would like to contribute to this repository, please discuss the changes you would like to make by creating an issue.
```

Comment:
Should you add steps of how to create an issue?

Reply:
I will add a hyperlink to GitHub's tutorial on creating issues.

Reply:
Update: The hyperlink has been added